projects/examples/src/components/form-input/form-select.example.component.ts

```diff
@@ -33,6 +33,11 @@ export class FormSelectExampleComponent {
             display: 'option3',
```

Comment:
Can we create a new example so that each example can show a different feature? We try to avoid showing multiple features per example. Also, there should be some text over the example explaining the feature being exemplified.

We have a schematic that makes it easier to create a new example